### PR TITLE
Twice decorator

### PR DESCRIPTION
**Pull request by eliacharfe**
Pull request - twice decorator.

### CODE REVIEW

File: twice_decorator.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+from pyclbr import Function
+import decorator
+
+
+class CustomError(Exception):
```
Comment (talbarfm) [inline twice_decorator.py]: What is this exception and why is this here? it is not used

File: twice_decorator.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+
+def twice_decorator(func: Function):
+    """
+    A decorator that executes the function it wraps twice.
+
+    :param func: The function that have the property @twice_decorator that been called.
+    :return: Decorator of the inner function.
+    """
+    def wrapped_function(function, argument) -> int:
```
Comment (talbarfm) [inline twice_decorator.py]: Use decorator.decorator as a decorator, not as a function, to reduce boilerplate code

File: twice_decorator.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
+from pyclbr import Function
+import decorator
+
+
+class CustomError(Exception):
+    pass
+
+
+def twice_decorator(func: Function):
```
Comment (talbarfm) [inline twice_decorator.py]: What about return types

**Review by talbarfm — CHANGES_REQUESTED** [review]

File: twice_decorator.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    :return: The integer sent multiply by 2.
+    """
+    print("in times2 function")
+    return num * 2
+
+
+if __name__ == '__main__':
+    try:
+        times2(5)
+    except Exception as e:
```
Comment (talbarfm) [inline twice_decorator.py]: why ?

File: twice_decorator.py (hunk 2 above)
Comment (eliacharfe) [reply twice_decorator.py]: I don't understand this remark - I used decorator.decorator...

Comment (talbarfm) [reply twice_decorator.py]: Look how it is used in the official docs - you dont use it right. 
https://github.com/micheles/decorator/blob/master/docs/documentation.md
Also, look the the implementation of decorator.decorator - especially the second argument you are giving it

**Review by talbarfm — CHANGES_REQUESTED** [review]

**Comment by eliacharfe** [comment]
I updated the exercise, can you tell me if I used decorator.decorator in the right way?
